packages/custom-url/src/Infrastructure/Symfony/Router/CustomUrlRouter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\CustomUrl\Infrastructure\Symfony\Router;
+
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\RequestContext;
+use Symfony\Component\Routing\RouteCollection;
+use Symfony\Component\Routing\RouterInterface;
+
+class CustomUrlRouter implements RouterInterface
```

Comment:
this will be refactored to our new routing providers. currently  out of scope until we removed all old routing parts

Reply:
@alexander-schranz should we then also remove this?

Reply:
yes router should be removed the website requires a new implementation on our own:  `RouteCollectionForRequestLoaderInterface` by add a new service to the `Infrastructure\Sulu\Route`